Marlin/Configuration_adv.h

```diff
@@ -1531,6 +1531,7 @@
       #define BOOTSCREEN_TIMEOUT 4000       // (ms) Total Duration to display the boot screen(s)
       #if EITHER(HAS_MARLINUI_U8GLIB, TFT_COLOR_UI)
         #define BOOT_MARLIN_LOGO_SMALL      // Show a smaller Marlin logo on the Boot Screen (saving lots of flash)
+        //#define BOOT_MARLIN_LOGO_ANIMATED   // Choose either small static logo, large animated logo, or BOTH shows small animated logo
```

Comment:
This option already exists:

https://github.com/MarlinFirmware/Marlin/blob/161b99cb54f5d6d183877902581d3d77e8a1fa78/Marlin/Configuration_adv.h#L1950

Reply:
if `BOOT_MARLIN_LOGO_ANIMATED` already exists, then I believe that or `BOOT_MARLIN_LOGO_SMALL` be relocated.

as I've said, it isn't clear that having one or the other or both enabled each does something different. with that fact in mind, `BOOT_MARLIN_LOGO_ANIMATED` be relocated next to the very same option except for the last word in the definition - "_SMALL" + "_ANIMATED" . they both link to dogm_Bootscreen.h and the code they enable is close in proximity, so it only makes sense to bring these two options to together since they are indeed related and dependant with another.